bindings/rust/evmc-vm/src/lib.rs

```diff
@@ -2,6 +2,179 @@ pub extern crate evmc_sys;
 pub use evmc_sys as ffi;
 
 // TODO: Add helpers for host interface
-// TODO: Add convenient helpers for evmc_result
 // TODO: Add convenient helpers for evmc_execute
 // TODO: Add a derive macro here for creating evmc_create
+
+/// EVMC result structure.
+pub struct ExecutionResult {
+    status: ffi::evmc_status_code,
+    gas: i64,
+    output: Option<Vec<u8>>,
+    release_fn: Option<unsafe extern "C" fn(*const ffi::evmc_result)>,
```

Comment:
Isn't there type for this generated by the bindgen?

Found it, `evmc_release_result_fn`.

Reply:
But actually this member is not needed.